### PR TITLE
 chore: change the display text for wake up setting 

### PR DESCRIPTION
**Pull request by Decodetalkers**
Log:

### CODE REVIEW

**Comment by Decodetalkers** [comment]
https://cooperation.uniontech.com/app/6f20cf83-ef8a-4067-bac9-b05c6d6bbcb6/62d775ebb700f345f3f9705d/650800e20a1f50226ade22ca/650800e20a1f50226ade22cb 这个需求，需要添加翻译

File: src/plugin-power/window/generalmodule.cpp [hunk 1]
```diff
@@ -322,7 +322,7 @@ void GeneralModule::initUI()
     appendChild(group);
     group->appendChild(new ItemModule(
             "passwordIsRequiredToWakeUpTheComputer",
-            tr("Password is required to wake up the computer"),
+            tr("Doing unlock step is required to wake up the computer"),
```
Comment (BLumia) [inline src/plugin-power/window/generalmodule.cpp:325]:
```suggestion
            tr("Unlocking is required to wake up the computer"),
```

下同

**Review by BLumia — DISMISSED** [review]

**Review by BLumia — APPROVED** [review]

**Pull request merged**